scipy/signal/filter_design.py

```diff
@@ -4843,6 +4845,203 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
+    """
+    Utility for converting from frequency (Hz) to the
+    Equivalent Rectangular Bandwith (ERB) scale
+    """
+    EarQ = 9.26449
+    minBW = 24.7
+    order = 1
+    return ((hz / EarQ) ** order + minBW ** order) ** (1 / order)
+
+
+def gammatone(freq, ftype, order=None, numtaps=None, fs=None):
+    """
+    Gammatone filter design.
+
+    This function computes the coefficients of an FIR or IIR gammatone
+    digital filter [1].
+
+    Parameters
+    ----------
+    freq : float
+        Center frequency of the filter (expressed in the same units
+        as `fs`).
```

Comment:
What's the unit when fs is not given since it is optional?

Reply:
The default value for fs is 2, like firwin() and iirfilter(). Added it in the docstring.

Reply:
But what is the unit, 2 hz.?

Reply:
Please refer to an earlier comment on this. Also, in firwin() it does not specify the unit as Hz for fs.
https://github.com/scipy/scipy/pull/12200#discussion_r430610263